src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -309,8 +408,13 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, OrgPersonMix
 
   // Store getter
   @Getter getOrgPeople!: OrgPersonIF[]
+  @Getter isAlterationFiling!: boolean
   @Getter isBenCorrectionFiling!: boolean
+  @Getter isCorrectionFiling!: boolean
+  @Getter isFirmChangeFiling!: boolean
+  @Getter isFirmConversionFiling!: boolean
   @Getter isFirmCorrectionFiling!: boolean
+  @Getter isRoleStaff!: boolean
```

Comment:
Vetur extension said these weren't needed because they were imported from a mixin, but Volar says they're not found otherwise. I think it works both ways but I prefer fewer warnings :)